### DISCUSSION TITLE
Backup Config

### DISCUSSION THREAD

**Discussion by dieBakterie** [prasanthrangan/hyprdots#1257] [Q&A]
How do i add config files to the backup list, i don't know how exactly this is handled? I've added the following files

1. `~/.config/hypr/hyprlock.conf`
2. `~/.config/hypr/hypridle.conf`
3. `~/.config/hypr/launch.conf`

and edited another ones and i want to backup all files in `~/.config/hypr/` before installating hyde. Sorry if there is already a discussion about this.

**Comment by rubiin** [comment]
+1
